app/experimenter/experiments/models/nimbus.py

```diff
@@ -20,6 +20,14 @@ class NimbusExperimentManager(models.Manager):
     def launch_queue(self, application):
         return self.filter(status=NimbusExperiment.Status.REVIEW, application=application)
 
+    def pause_queue(self, application):
+        return self.filter(
+            id__in=[
+                experiment.id for experiment in self.all() if experiment.should_pause
+            ],
+            application=application,
```

Comment:
Oh yeah we also gotta filter for `status` and `is_paused` here, from the previous task:

```py
        live_experiments = NimbusExperiment.objects.filter(
            status=NimbusExperiment.Status.LIVE,
            application=application,
            is_paused=False,
        )
```